php/lang/security/audit/assert-use-audit.yaml

```diff
@@ -3,6 +3,9 @@ rules:
   patterns:
   - pattern: assert($ASSERT, ...);
   - pattern-not: assert("...", ...);
+  - metavariable-regex:
+      metavariable: $ASSERT
+      regex: \A\$\S+\Z
```

Comment:
This looks for non-whitespace, right? Would an expression without whitespace be handled correctly? For example

    assert($ok!="something");

Reply:
it would not. Updated regex to be more opinionated - there will likely still be edge cases that will be inappropriately flagged but as this is an audit rule and not meant for CI, there should be a meaningful result improvement.